tests/test_docker_compose.py

```diff
@@ -23,6 +25,18 @@ def test_can_pull_images_before_spawning_service_via_compose():
         assert port == "4444"
 
 
+def test_can_build_images_before_spawning_service_via_compose():
+    with patch.object(DockerCompose, DockerCompose._call_command.__name__) as call_mock:
+        with DockerCompose("tests", build=True) as compose:
+            ...
+
+    assert compose.build
```

Comment:
This is an implementation detail. There is no need to test for this attribute because passing this attribute is tested below

Reply:
It depends on if you see the attribute as a feature for users (i.e. you want users to have `compose.build == True` if it's passed at construction).  

But again, I don't feel strongly about it so I'll delete it.

Reply:
Hm, you convinced me. Let's leave it like this ;)